akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -639,7 +639,6 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
           val map = singleMapEntryFromProto(entry.getEntryData, (v: dm.OtherMessage) ⇒ otherMessageFromProto(v).asInstanceOf[ReplicatedData])
           ORMap.PutDeltaOp(ORSet.AddDeltaOp(orsetFromProto(entry.getUnderlying)), map.head, zeroTagFromCode(entry.getZeroTag))
         } else if (entry.getOperation == rd.ORMapDeltaOp.ORMapRemove) {
-          val map = singleMapEntryFromProto(entry.getEntryData, (v: dm.OtherMessage) ⇒ otherMessageFromProto(v).asInstanceOf[ReplicatedData])
           ORMap.RemoveDeltaOp(ORSet.RemoveDeltaOp(orsetFromProto(entry.getUnderlying)), zeroTagFromCode(entry.getZeroTag))
         } else if (entry.getOperation == rd.ORMapDeltaOp.ORMapRemoveKey) {
           val map = singleMapEntryFromProto(entry.getEntryData, (v: dm.OtherMessage) ⇒ otherMessageFromProto(v).asInstanceOf[ReplicatedData])
```

Comment:
@gosubpl There is also a
```
java.lang.ClassCastException: java.lang.String cannot be cast to akka.cluster.ddata.ReplicatedData
```
thrown from above line. I'm a bit lost in all the casts here, perhaps you can see what is wrong.

The full stack trace is:
```
: Error [java.lang.String cannot be cast to akka.cluster.ddata.ReplicatedData] [
[JVM-2] java.lang.ClassCastException: java.lang.String cannot be cast to akka.cluster.ddata.ReplicatedData
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer$$anonfun$40$$anonfun$42.apply(ReplicatedDataSerializer.scala:644)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer$$anonfun$40$$anonfun$42.apply(ReplicatedDataSerializer.scala:644)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer.singleMapEntryFromProto(ReplicatedDataSerializer.scala:577)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer$$anonfun$40.apply(ReplicatedDataSerializer.scala:644)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer$$anonfun$40.apply(ReplicatedDataSerializer.scala:637)
[JVM-2] 	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
[JVM-2] 	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
[JVM-2] 	at scala.collection.Iterator$class.foreach(Iterator.scala:893)
[JVM-2] 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1336)
[JVM-2] 	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
[JVM-2] 	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
[JVM-2] 	at scala.collection.TraversableLike$class.map(TraversableLike.scala:234)
[JVM-2] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer.akka$cluster$ddata$protobuf$ReplicatedDataSerializer$$ormapDeltaGroupFromBinary(ReplicatedDataSerializer.scala:651)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer$$anonfun$16.apply(ReplicatedDataSerializer.scala:226)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer$$anonfun$16.apply(ReplicatedDataSerializer.scala:226)
[JVM-2] 	at akka.cluster.ddata.protobuf.ReplicatedDataSerializer.fromBinary(ReplicatedDataSerializer.scala:312)
```

I debugged that it is a "f" String that is deserialized via `otherMessageFromProto`. I think that is a key.